src/holofuel/pages/TransactionHistory/TransactionHistory.js

```diff
@@ -123,6 +128,8 @@ export default function TransactionsHistory () {
       </React.Fragment>)}
     </div>}
 
+    {hasMoreTransactions && <Button onClick={showMoreTransactions}>Load More</Button>}
```

Comment:
Let's add the conditional that omits this button component whenever there are no transactions.
ie: `{!noVisibleTransactions && hasMoreTransactions && <Button onClick={showMoreTransactions}>Load More</Button>}`

Reply:
awkwardly, it's possible for there to be more transactions to load, even while there are `noVisibleTransactions`. (eg: there are no withdrawals in the current transaction list, but there are some later.) This is a slightly broken UX, and we can make it smarter, but I want to get a first iteration in before we ship.